virt/install/preparing-cluster-for-virt.adoc

```diff
@@ -60,7 +60,7 @@ Live migration has the following requirements:
 
 * Shared storage with `ReadWriteMany` (RWX) access mode
 * Sufficient RAM and network bandwidth
-* Appropriate CPUs with sufficient capacity on the worker nodes. If the CPUs have different capacities, live migration might be very slow or fail.
+* Node support for host model CPU. If the virtual machine uses a host model CPU, live migration is supported only between nodes that support the virtual machine's host model CPU.
```

Comment:
```suggestion
* If the virtual machine uses a host model CPU, live migration is supported only between nodes that support the virtual machine's host model CPU.
```
If the VM uses a specific model instead of variable host-model, then this is not a requirement.